gallery_dl/extractor/photovogue.py

```diff
@@ -0,0 +1,70 @@
+# -*- coding: utf-8 -*-
+
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License version 2 as
+# published by the Free Software Foundation.
+
+"""Extractors for https://www.vogue.it/en/photovogue/"""
+
+from .common import Extractor, Message
+from datetime import datetime
+
+BASE_PATTERN = r"(?:https?://)?(?:www.vogue.it(?:/en)?/photovogue)"
+
+
+class PhotovogueUserExtractor(Extractor):
+    category = "photovogue"
+    subcategory = "user"
+    directory_fmt = ("{category}", "{photographer[id]}_{photographer[name]}")
+    filename_fmt = "{id}_{title}.{extension}"
+    archive_fmt = "{id}"
+    root = "https://www.vogue.it/en/photovogue/"
+    pattern = BASE_PATTERN + r"/portfolio/\?id=(\d+)"
+    test = (
+        ("https://www.vogue.it/en/photovogue/portfolio/?id=221252",),
+        ("https://www.vogue.it/photovogue/portfolio/?id=221252",),
```

Comment:
```suggestion
        ("https://www.vogue.it/en/photovogue/portfolio/?id=221252"),
        ("https://vogue.it/photovogue/portfolio/?id=221252"),
```
Tests should either be a single string or a (url, results) tuple.
`python test_results.py photovogue` fails when they are single-element tuples.